docs/local-development.md

```diff
@@ -12,13 +12,21 @@ The airplay receiver uses port 5000, which may need to be disabled. Further deta
 
 # Prerequisites
 
-In order to run through this example, you will need the following installed:
-
- * Docker
+In order to run through this example, you will need the following installed. For
+Windows installation, the tools will get installed in the WSL (see link below)
+and in particular, I tested this with installing Ubuntu 20.04 LTS. Then all
+the commands are executed in that environment. Also for Windows, you will need
+to install Windows Docker desktop after which you will need to install Docker
+cli on the Linux subsystem.
+
+ * [Docker](https://docs.docker.com/get-docker/)
```

Comment:
most of these are the same for everyone, no matter runtime or os, when i was updating the GCP tutorial I moved this into the README.md.

Reply:
Ok. Do you want to merge this and then once your PR lands, remove them from here? Or what do you think?